optaplanner-benchmark/src/test/java/org/optaplanner/benchmark/impl/measurement/ScoreDifferencePercentageTest.java

```diff
@@ -0,0 +1,192 @@
+package org.optaplanner.benchmark.impl.measurement;
+
+import org.junit.Test;
+import org.optaplanner.core.api.score.buildin.bendable.BendableScore;
+import org.optaplanner.core.api.score.buildin.hardsoft.HardSoftScore;
+import org.optaplanner.core.api.score.buildin.simple.SimpleScore;
+
+import static org.junit.Assert.assertEquals;
+
+public class ScoreDifferencePercentageTest {
+
+    @Test(expected = IllegalStateException.class)
+    public void calculateScoreDifferencePercentageException() {
+        int[] hardScore1 = {1, 2, 3};
+        int[] softScore1 = {4, 5};
+        int[] hardScore2 = {1, 2};
+        int[] softScore2 = {4, 5};
+        BendableScore score1 = BendableScore.valueOf(hardScore1, softScore1);
+        BendableScore score2 = BendableScore.valueOf(hardScore2, softScore2);
+        ScoreDifferencePercentage.calculateScoreDifferencePercentage(score1, score2);
+    }
+
+    @Test
+    public void calculateScoreDifferencePercentage() {
+        double tolerance = 0.00001;
+        SimpleScore score1 = SimpleScore.parseScore("-100");
```

Comment:
Prefer SimpleScore.valueOf(-100); instead in future test cases (I 'll change that for this one)